pkg/util/mount/nsenter_mount_test.go

```diff
@@ -227,8 +228,26 @@ func TestNsenterExistsFile(t *testing.T) {
 
 				return path, nil
 			},
-			expectedOutput: false,
-			expectError:    true,
+			expectedOutput: func() bool {
+				user, err := user.Current()
```

Comment:
Can you detect that the user is root somewhere above (i.e. at the beginning of `TestNsenterExistsFile`) once and use it here?

```go
expectedOutput: isRoot, // ExistsPath succeeds when running as root
expectedError: !isRoot, // ExistsPath must fail when running as non-root
```